server/src/logic/controller/formater.js

```diff
@@ -1,113 +1,96 @@
+  etablissement_formateur_sirenetablissement_reference: 1,
+  etablissement_gestionnaire_siretetablissement_reference: 1,
+  etablissement_gestionnaire_enseigneetablissement_reference: 1,
+  etablissement_gestionnaire_uaietablissement_reference: 1,
+  etablissement_gestionnaire_adresseetablissement_reference: 1,
+  etablissement_gestionnaire_code_postaletablissement_reference: 1,
+  etablissement_gestionnaire_localiteetablissement_reference: 1,
+  etablissement_gestionnaire_complement_adresseetablissement_reference: 1,
+  etablissement_gestionnaire_cedexetablissement_reference: 1,
+  etablissement_gestionnaire_entreprise_raison_socialeetablissement_reference: 1,
+  etablissement_gestionnaire_sirenetablissement_reference: 1,
+  nom_academieetablissement_reference: 1,
+  num_academieetablissement_reference: 1,
+  code_postaletablissement_reference: 1,
+  code_commune_inseeetablissement_reference: 1,
+  num_departementetablissement_reference: 1,
+  uai_formationetablissement_reference: 1,
+  nometablissement_reference: 1,
+  intitule_longetablissement_reference: 1,
+  intitule_courtetablissement_reference: 1,
+  diplomeetablissement_reference: 1,
+  niveauetablissement_reference: 1,
+  cfdetablissement_reference: 1,
+  mef_10_codeetablissement_reference: 1,
+  rncp_codeetablissement_reference: 1,
+  rncp_intituleetablissement_reference: 1,
+  parcoursup_referenceetablissement_reference: 1,
+  parcoursup_a_chargeretablissement_reference: 1,
+  affelnet_referenceetablissement_reference: 1,
+  affelnet_a_chargeretablissement_reference: 1,
+  _idetablissement_reference: 1,
+  etablissement_formateur_idetablissement_reference: 1,
+  etablissement_gestionnaire_idetablissement_reference: 1,
 };
 
-const formatMEF = (mef) => {
-  const mef10 = mef.slice(0, -1);
-  const isValid = isFinite(parseInt(mef10));
-
-  if (!isValid) return "";
-
-  return mef10;
+const etablissement = {
+  siege_social: 1,
+  etablissement_siege_siret: 1,
+  siret: 1,
+  siren: 1,
+  naf_code: 1,
+  naf_libelle: 1,
+  date_creation: 1,
+  date_mise_a_jour: 1,
+  enseigne: 1,
+  adresse: 1,
+  numero_voie: 1,
+  type_voie: 1,
+  nom_voie: 1,
+  complement_adresse: 1,
+  code_postal: 1,
+  num_departement: 1,
+  localite: 1,
+  code_insee_localite: 1,
+  cedex: 1,
+  date_fermeture: 1,
+  ferme: 1,
+  region_implantation_code: 1,
+  region_implantation_nom: 1,
+  commune_implantation_code: 1,
+  commune_implantation_nom: 1,
+  num_academie: 1,
+  nom_academie: 1,
+  uai: 1,
+  entreprise_siren: 1,
+  entreprise_enseigne: 1,
+  entreprise_raison_sociale: 1,
+  entreprise_nom_commercial: 1,
+  entreprise_date_creation: 1,
+  entreprise_date_radiation: 1,
+  entreprise_siret_siege_social: 1,
+  raison_sociale: 1,
+  nom_commercial: 1,
+  siret_siege_social: 1,
+  uais_formations: 1,
+  created_at: 1,
+  last_update_at: 1,
+  entreprise_tranche_effectif_salarie: 1,
+  etablissement_siege_id: 1,
+  matched_uai: 1,
+  id_mna_etablissement: 1,
```

Comment:
this field doesn't exist in the `Etablissement` Model. You may select the `_id` and add it when you push the object in your data :)